tests/mcs/resource_manager/resource_manager_test.go

```diff
@@ -128,6 +128,9 @@ func (suite *resourceManagerClientTestSuite) cleanupResourceGroups() {
 	groups, err := cli.ListResourceGroups(suite.ctx)
 	suite.NoError(err)
 	for _, group := range groups {
+		if group.Name == "default" {
```

Comment:
Can we refuse to delete or modify the default resource group in `manager.go`?

Reply:
It's already blocked at TiDB side.

Reply:
I also added in pd side.  ptal @CabinfeverB